roles/reset/tasks/main.yml

```diff
@@ -381,6 +381,8 @@
   include_role:
     name: "network_plugin/{{ kube_network_plugin }}"
     tasks_from: reset
+  vars:
```

Comment:
Calling the reset task should not concern any dependencies of the called roles. If CNI plugins need to be reset then the call should be made for `network_plugin/cni` and `network_plugin/{{ kube_network_plugin }}`